### ISSUE TITLE
Update documentation on the league website to match the README on the ~3.0 branch

### ISSUE THREAD

**Issue by ctrlaltdylan** [thephpleague/factory-muffin#448]
Not sure the existing deployment method from markdown to html on the league website, but the current site on the league website is showing documentation for v2. This is pretty confusing because it heavily relies on a Facade class that I _think_ is no longer in use in v3.

**Comment by igneus** [comment]
Duplicate of #362

**Comment by GrahamCampbell** [comment]
> Duplicate of #362

No it's not. #362 is specifically about the upgrading guide not having as much detail for the last release as was the case with previous major releases.

**Comment by igorsantos07** [comment]
Got fooled by that comment...
But yeah, 4~5 years of a release without updated docs is pretty lame IMHO. The entire website is outdated, so until this is solved by either maintainers or a PR, I would suggest making it redirect to the README instead.

**Comment by GrahamCampbell** [comment]
The website is open source, and open to PRs.

**Comment by igorsantos07** [comment]
Yeah, for 3 years. If you're not going to fix your own release, and I don't have the time to give you that (otherwise I would have already, you've _already_ suggested that I should), just redirect it. Stop fooling others into the same trap.

**Comment by GrahamCampbell** [comment]
Hardly a trap. v2 still exists, and you can use it exactly how the website says. It even tells you how to install it.

**Comment by igorsantos07** [comment]
So the suggestion is to use an old (unmaintained?) version just because
that's the one with documentation up to date, even if the next version was
released 5 years ago?

Seriously, what's wrong with getting the website down instead of leaving it
outdated forever there? If you're not going to update it, remove. README is
enough and we're used to reading long README files, just don't leave
misleading stuff on the web.

**Comment by GrahamCampbell** [comment]
I suppose I could also test version 2.1.x against PHP 7. It should work just fine.
